test/dlc_tests/conftest.py

```diff
@@ -409,6 +412,7 @@ def framework_version_within_limit(metafunc_obj, image):
 def pytest_configure(config):
     # register canary marker
     config.addinivalue_line("markers", "canary(message): mark test to run as a part of canary tests.")
+    config.addinivalue_line("markers", "quick_check(message): mark test to run as a part of quick check tests.")
```

Comment:
Are there any other test in sanity test that qualifies for quick_check test?

Reply:
Also going to add git secrets test